### ISSUE TITLE
[Accessibility]: AVT errors are coming for the section of graphs and charts where download, expand and export icons are there..

### ISSUE THREAD

**Issue by priyanka1304** [carbon-design-system/carbon-charts#1348]
### Contact Details

prisi174@in.ibm.com

### Environment

Browser

### What happened? What did you expect to happen instead?

I am getting following AVT error for the highlighted parts of charts shown below.
<img width="540" alt="Screenshot 2022-04-08 at 11 47 57 AM" src="https://user-images.githubusercontent.com/35679136/162688123-0459b472-a9d2-469c-b8d3-bece2bc4140b.png">
<img width="938" alt="Screenshot 2022-04-08 at 11 53 06 AM" src="https://user-images.githubusercontent.com/35679136/162688136-64eeba2b-b3de-48ae-8e66-dd78e8c447af.png">

### What WCAG 2.1 checkpoint does the issue violate?

Checkpoint 4.1.2: Name, Role, Value Level (A)

### Version

@carbon/charts": "0.55.1"

### Codesandbox example

https://codesandbox.io/s/affectionate-bird-ziwg8l?file=/src/index.js

**Comment by theiliad** [comment]
I believe fix was pushed 4 days ago
